combirepo/rpm_patcher.py

```diff
@@ -648,22 +648,19 @@ def __postprocess_cache(self):
             info = "{0}".format(matching_task)
             with open(info_path, "wb") as info_file:
                 info_file.write(info)
-            copy_tasks.append((name, path, destination_path))
-        hidden_subprocess.function_call_list(
-            "Copying to cache", shutil.copy, copy_tasks)
+            hidden_subprocess.call("Copying to cache",
+                                   ["sudo", "cp", path, destination_path])
 
     def __process_results(self):
         """
         Processes final results of patcher.
         """
         results = self._get_results()
-        copy_tasks = []
         for info in results:
             name, path, _ = info
             target = self._targets[name]
-            copy_tasks.append((name, path, target))
-        hidden_subprocess.function_call_list(
-            "Copying to repo", shutil.copy, copy_tasks)
+            hidden_subprocess.call("Copying to repo",
```

Comment:
The same question about performance

Reply:
It takes about minute to copy one thousand of packages this way.

Reply:
I tried to pass all packages in one copy command but it overfilled args length.